packages/editor/src/client/panel/userEditPanel.tsx

```diff
@@ -67,8 +67,8 @@ function updateAddressObject(
 
 export function UserEditPanel({id, onClose, onSave}: UserEditPanelProps) {
   const [name, setName] = useState('')
-  const [firstName, setFirstName] = useState<string | undefined>()
-  const [preferredName, setPreferredName] = useState<string | undefined>()
+  const [firstName, setFirstName] = useState('')
```

Comment:
Wasn't this for a reason `undefined` instead of empty `string`?

Reply:
`firstName` is only used in the formControl in this component. React disallows undefined to string values in form controls, or rather console.errors about it. As they go from uncontrolled to controlled.

Reply:
This is for example what is happening on master on the edit user panel:

![Screenshot 2022-05-09 at 11 12 42](https://user-images.githubusercontent.com/1131901/167379718-024fb044-a541-45af-ae97-17fcc771b2f8.png)